discretenet/problems/gisp/gisp.py

```diff
@@ -0,0 +1,216 @@
+from pathlib import Path
+import random
+from typing import Union
+import pyomo.environ as pyo
+import networkx as nx
+try:
+    from importlib.resources import open_text
+except ImportError:
+    from importlib_resources import open_text
+
+from discretenet.problem import Problem
+from discretenet.generator import Generator
+
+
+class GISPProblem(Problem):
+    def __init__(self, graph, E2, name):
```

Comment:
Even though it's empty, you should still call `super().__init__()` at the top of the subclass init method

Reply:
Updated